targets/ipq40xx-mikrotik

```diff
@@ -0,0 +1,6 @@
+include 'mikrotik.inc'
+
+device('mikrotik-sxtsq-5-ac', 'mikrotik_sxtsq-5-ac', {
```

Comment:
```suggestion
device('mikrotik-sxtsq-5-ac-rbsxtsqg-5acd', 'mikrotik_sxtsq-5-ac', {
```

Reply:
This should result in a matching string referencing the [modelname](https://github.com/openwrt/openwrt/blob/ebfe66e494e57f4b421f1190d6bff1d361db1b3d/target/linux/ipq40xx/image/mikrotik.mk#L60) from OpenWrt.